src/BusinessLogic/Bootstrap/Aspect/Aspects.php

```diff
@@ -47,29 +52,49 @@ public function andRun(Aspect $aspect): self
     /**
      * @param T $subject
      *
+     * // @phpstan-ignore-next-line
+     *
      * @return T
      */
     public function beforeEachMethodOfInstance($subject)
     {
         $this->subject = $subject;
         $this->subjectClassName = null;
 
+        /**
+ * @noinspection PhpIncompatibleReturnTypeInspection
+*/
+        // @phpstan-ignore-next-line
```

Comment:
```suggestion
        /**
         * @noinspection PhpIncompatibleReturnTypeInspection
        */
        // @phpstan-ignore-next-line
```

What is the issue here?